### PR TITLE
#367 Create Medium Polymer Item

### PR DESCRIPTION
**Pull request by ryanquincypaul**
Issue: https://github.com/Menacing/unnamed-extraction-shooter-roguelike/issues/367

### CODE REVIEW

**Comment by ryanquincypaul** [comment]
I named this Medium Polymer. Do we want it to be "medium polymer pile"? How is this material going to be used and is it a different thing from polymer_pile or is it just going to be like something you need less of (compared to the small pile) in the same crafts?

**Comment by Menacing** [comment]
> I named this Medium Polymer. Do we want it to be "medium polymer pile"? How is this material going to be used and is it a different thing from polymer_pile or is it just going to be like something you need less of (compared to the small pile) in the same crafts?

Yeah, 3 sizes, all the same material, different values and inventory sizes
https://docs.google.com/document/d/14sWBimzbu8NZH2ulE1exa6A2DPjcmRtdcEeepyEGrHU/edit

File: game_objects/items/materials/medium_polymer/medium_polymer_inventory_info.tres [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+icon = ExtResource("3_df7ph")
+icon_r = ExtResource("3_df7ph")
+detail_scene = ExtResource("3_ha2yn")
+context_menu_items = Array[Resource("res://ui/inventory/item_context_item.gd")]([ExtResource("1_2vlxm"), ExtResource("1_efr8v")])
+max_stacks = 10
+has_stacks = true
+max_durability = 1
+has_durability = false
+column_span = 2
+row_span = 1
```
Comment (Menacing) [inline game_objects/items/materials/medium_polymer/medium_polymer_inventory_info.tres:29]: should be 2

**Comment by Menacing** [comment]
Other than the item inventory size, looks great!

**Review by Menacing — APPROVED** [review]
Great job!

**Pull request merged**